src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,106 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import './FindMovie.scss';
-
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getMovie } from '../../api/movies';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie, setError, error }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState('');
+
+  const handleSearch = ({ target }) => {
+    setError(false);
+    setQuery(target.value);
+  };
+
+  const searchBtnHandler = () => {
```

Comment:
add this handler on submit, so users can search for the movie using the enter button